data/data/agent/files/usr/local/bin/common.sh

```diff
@@ -21,8 +22,8 @@ curl_assisted_service() {
             ;;
         "GET")
             curl -s -S -X GET "${additional_options[@]}" "${baseURL}${endpoint}" \
-            -H "Authorization: ${AGENT_AUTH_TOKEN}" \
```

Comment:
The PATCH doesn't need the ${authz} token?

Reply:
The changes for `PATCH` got added here via rebase and originally came from a commit on Sep 30 i.e. https://github.com/openshift/installer/commit/faaddc162915d15ae2df2368e2ab7a1f3af22144 however I think its a wrong commit because `PATCH` is not used in any curl requests in installer repo. The correct use is in appliance [here](https://github.com/openshift/appliance/blob/1c405b5cc722b29edcf4bb6bbe14e44d21a4c066/data/scripts/bin/common.sh.template#L10-L14) and [here](https://github.com/openshift/appliance/blob/1c405b5cc722b29edcf4bb6bbe14e44d21a4c066/data/scripts/bin/update-hosts.sh.template#L29-L30)

Reply:
Right, for now the appliance isn't directly using the code (the func was copied to the codebase instead).
Will probably be handled later on to reuse this func, so I think it'd be safer to update the PATCH flow as well.

Reply:
ok, I will update the `PATCH` flow here(even if its unused now). Please note , later in the appliance, you will need to pass the `USER_AUTH_TOKEN`